app/src/test/java/za/co/dvt/android/testing/RxViewModelTest.kt

```diff
@@ -43,9 +43,8 @@ class RxViewModelTest {
         doNothing().whenever(view).showSuccessfulMessage()
 
         val actual = systemUnderTest.fetchPlants().test()
-
-        val result = actual.events.first().map { it }.first() as ArrayList<Plant>
-        assert(result.isEmpty())
+        val result = actual.values()
+        assert(result.first().isEmpty())
```

Comment:
Shouldn't this be `assertTrue()`?
```suggestion
        assertTrue(result.first().isEmpty())
```

Reply:
Assert and AssertTrue is the same thing 😀 if you think being more explicit is better I don't mind changing it